enclave-tls/src/Makefile

```diff
@@ -1,18 +1,21 @@
-TOPDIR ?= ..
+TOPDIR ?= $(shell readlink -f ..)
 
 include $(TOPDIR)/common.mk
 
 LIBDIR := $(TOPDIR)/src/lib
 TLS_WRAPPERS := $(LIBDIR)/tls_wrappers
 ENCLAVE_QUOTES := $(LIBDIR)/enclave_quotes
 INCDIR := $(TOPDIR)/src/include
+WOLFSSL_ROOT := $(TOPDIR)/src/wolfssl
+WOLFSSL_BUILD := $(LIBDIR)/.build
```

Comment:
Looks like it is necessary to move these stuffs to tls wrapper specific makefile rather than the top makefile.